processor/transformprocessor/README.md

```diff
@@ -1,10 +1,11 @@
 # Transform Processor
 
-| Status                   |                       |
-| ------------------------ | --------------------- |
-| Stability                | [alpha]               |
-| Supported pipeline types | traces, metrics, logs |
-| Distributions            | [contrib]             |
+| Status                   |                                                              |
+|--------------------------|--------------------------------------------------------------|
+| Stability                | [alpha]                                                      |
+| Supported pipeline types | traces, metrics, logs                                        |
+| Distributions            | [contrib]                                                    |
+| Warnings                 | [Identity Crisis, Unsound Transformations, Other](#warnings) |
```

Comment:
Note sure if we want to put everything here, but if we do the new one is missing
```suggestion
| Warnings                 | [Identity Crisis, Unsound Transformations, Orphaned Telemetry, Other](#warnings) |
```